src/index/base.cpp

```diff
@@ -18,8 +18,8 @@ using node::ReadBlockFromDisk;
 
 constexpr uint8_t DB_BEST_BLOCK{'B'};
 
-constexpr int64_t SYNC_LOG_INTERVAL = 30; // seconds
-constexpr int64_t SYNC_LOCATOR_WRITE_INTERVAL = 30; // seconds
+constexpr std::chrono::seconds SYNC_LOG_INTERVAL{30};
+constexpr std::chrono::seconds SYNC_LOCATOR_WRITE_INTERVAL{30};
```

Comment:
here and the line above and lines 133-134, can use [std::chrono_literals](https://en.cppreference.com/w/cpp/symbol_index/chrono_literals)
```suggestion
constexpr auto SYNC_LOCATOR_WRITE_INTERVAL{30s};
```

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/25079/commits/626b636b193baae81a697febd41a712f94a8a702. Thanks for suggestion.